k8s/modules_aws_snapshots_dev.tf

```diff
@@ -0,0 +1,23 @@
+module "test-calibrationapi" {
```

Comment:
do we need to have test word in the module name?

Reply:
1. Right now I've added only testing stuff here, I want to discuss how often we should do them and how many to keep, especially in mainnet.
2. After discussion I'll enable them for dev cluster node, check that everything is fine
3. Enable in mainnet